ghmirror/app/response.py

```diff
@@ -0,0 +1,44 @@
+class MirrorResponse:
+
```

Comment:
a bit of doc in this class would be helpful

Reply:
Yes,we need docstrings all around in fact (the pylint check is strategically disabled for now :)  )